crime-commons-mod-schemas/src/main/resources/schemas/ioj/apiPerformIojAppealRequest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "apiPerformIojAppealRequest.json",
+  "type": "object",
+  "title": "IoJ Appeal request",
```

Comment:
Really minor but would it be a bit clearer for the title to be "Perform IoJ Appeal request"?

Reply:
This is going to be used for both create and update (though if we end up dropping update then the schema will be renamed from `perform` to `create` anyway). I'm happy to update the title to include perform, though the intention of the schema is covered in plain English by the description 🙂

Reply:
Happy what whatever you think is best 👍 , it's just a minor thing that I thought I'd point out as the response references perform in title.